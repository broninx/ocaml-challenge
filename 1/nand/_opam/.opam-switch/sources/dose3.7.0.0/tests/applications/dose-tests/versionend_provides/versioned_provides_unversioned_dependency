Package: main
Version: 1
Architecture: toaster
Depends: v

Package: b
Version: 1
Architecture: toaster
Provides: v (=1)

