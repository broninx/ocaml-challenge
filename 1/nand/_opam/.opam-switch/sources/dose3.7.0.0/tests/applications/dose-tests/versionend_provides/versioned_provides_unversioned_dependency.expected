output-version: 1.2
report:
 -
  package: main
  version: 1
  architecture: toaster
  status: ok
  
 
background-packages: 1
foreground-packages: 1
total-packages: 2
broken-packages: 0
